www/rustup.css

```diff
@@ -95,12 +114,12 @@ body#idx p.other-platforms-help {
 }
 
 .instructions > * {
-    width: 45rem;
```

Comment:
We no longer need the fixed width. We’re using percentages for better responsiveness.